src/coreclr/jit/compiler.cpp

```diff
@@ -6205,6 +6205,12 @@ int Compiler::compCompile(CORINFO_MODULE_HANDLE classPtr,
         {
             instructionSetFlags.AddInstructionSet(InstructionSet_APX);
         }
+
+#elif defined(TARGET_RISCV64)
+
+        instructionSetFlags.AddInstructionSet(InstructionSet_Zbb);
+        instructionSetFlags.AddInstructionSet(InstructionSet_Zba);
```

Comment:
```suggestion
        if (JitConfig.EnableHWIntrinsic() != 0)
        {
            instructionSetFlags.AddInstructionSet(InstructionSet_RiscVBase);
        }
        if (JitConfig.EnableZba() != 0)
        {
            instructionSetFlags.AddInstructionSet(InstructionSet_Zba);
        }
        if (JitConfig.EnableZbb() != 0)
        {
            instructionSetFlags.AddInstructionSet(InstructionSet_Zbb);
        }
```
And add `EnableZb(a|b)` to jitconfigvalues.h, clrconfigvalues.h, and codeman.cpp?

Reply:
Okay 👍